manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with WinGet Releaser v2 using Komac v1.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: twpayne.chezmoi
+PackageVersion: 2.34.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chezmoi.exe
+  PortableCommandAlias: chezmoi
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.2/chezmoi_2.34.2_windows_amd64.zip
+  InstallerSha256: 452B053A995154315B3A95569F5ADFC5343C2C6F97A036CDAD426C3E50FDDBE6

```

Comment:
```suggestion
```

Reply:
Thank you very much for the review and the suggested changes.

Clearly there is a mismatch between the `Architecture` field (`x86` in this case) and the architecture in the zip file name (`amd64`). This error is also present in the other `InstallerUrl`s and `InstallSha256`s as you identify below.

This PR was automatically generated by https://github.com/vedantmgoyal2009/winget-releaser, so I will raise an issue with the automation.

Reply:
Opened https://github.com/vedantmgoyal2009/winget-releaser/issues/173.